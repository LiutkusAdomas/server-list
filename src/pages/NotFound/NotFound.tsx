import { Container } from '../../components/common/Container/Container';
import { Heading } from '../../components/common/Heading/Heading';

export const NotFound: React.FC = () => {
  return (
    <Container>
      <Heading>Sorry, there is nothing here :(</Heading>
    </Container>
  );
};
