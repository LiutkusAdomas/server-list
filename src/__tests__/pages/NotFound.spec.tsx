import { render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';
import { NotFound } from '../../pages/NotFound/NotFound';

describe('Not Found page', () => {
  it('should render a not found page', () => {
    const { asFragment } = render(<NotFound />);
    expect(asFragment).toMatchSnapshot();
  });
});
