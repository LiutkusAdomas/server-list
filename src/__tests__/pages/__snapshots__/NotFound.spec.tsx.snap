// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Not Found page > should render a not found page 1`] = `[Function]`;
